src/plugins/LADTable/components/LADRow.vue

```diff
@@ -110,24 +112,34 @@ export default {
     },
     methods: {
         updateValues(datum) {
-            let newTimestamp = this.formats[this.timestampKey].parse(datum),
-                limit;
-
-            if(this.shouldUpdate(newTimestamp)) {
-                this.timestamp = this.formats[this.timestampKey].parse(datum);
-                this.value = this.formats[this.valueKey].format(datum);
-                limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
-                if (limit) {
-                    this.valueClass = limit.cssClass;
-                } else {
-                    this.valueClass = '';
+            if(datum) {
+                let newTimestamp = this.getParsedTimestamp(datum),
+                    limit;
+
+                if(this.shouldUpdate(newTimestamp)) {
+                    this.timestamp = this.getParsedTimestamp(datum);
+                    this.value = this.formats[this.valueKey].format(datum);
+                    limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
+                    if (limit) {
+                        this.valueClass = limit.cssClass;
+                    } else {
+                        this.valueClass = '';
+                    }
                 }
+            } else {
+                this.resetValues();
             }
         },
         shouldUpdate(newTimestamp) {
-            return (this.timestamp === undefined) ||
-                (this.inBounds(newTimestamp) &&
-                newTimestamp > this.timestamp);
+            let newTimestampInBounds = this.inBounds(newTimestamp),
+                noExistingTimestamp = this.timestamp === undefined,
+                existingTimestampInBounds = this.inBounds(this.timestamp),
```

Comment:
Don't worry about whether the existing value is out of bounds. I think it will simplify things if we just always throw it away when the user changes bounds.